### ISSUE TITLE
component broken on quasar v1.5.0

### ISSUE THREAD

**Issue by nothingismagick** [quasarframework/app-extension-qactivity#13]
Did the slot just change?
```
client?f1f8:135 ./node_modules/@quasar/quasar-app-extension-qactivity/src/component/QActivity.js 38:7-11
"export 'default' (imported as 'slot') was not found in 'quasar/src/utils/slot.js'
```

```
vue.runtime.esm.js?2b0e:1888 TypeError: Object(...) is not a function
    at Proxy.render (QActivity.js?c654:36)
    at VueComponent.Vue._render (vue.runtime.esm.js?2b0e:3542)
    at VueComponent.updateComponent (vue.runtime.esm.js?2b0e:4060)
    at Watcher.get (vue.runtime.esm.js?2b0e:4473)
    at new Watcher (vue.runtime.esm.js?2b0e:4462)
    at mountComponent (vue.runtime.esm.js?2b0e:4067)
    at VueComponent.Vue.$mount (vue.runtime.esm.js?2b0e:8409)
    at init (vue.runtime.esm.js?2b0e:3118)
    at createComponent (vue.runtime.esm.js?2b0e:5972)
    at createElm (vue.runtime.esm.js?2b0e:5919)
```

This return is the offending line.
```
  render (h) {
    return h('ul', {
      staticClass: 'q-activity',
      class: this.classes,
      style: this.styles
    }, slot(this, 'default'))
  }
```

**Comment by hawkeye64** [comment]
Fixed in 1.0.0-beta.4

**Comment by nothingismagick** [comment]
Great!
